docs/kqueen.rst

```diff
@@ -111,7 +113,7 @@ Example configuration files are located in `config/` directory. Default configur
       - Name of Jenkins job used to deploy cluster.
     * - JENKINS_USERNAME
       - None
-      - Username for Jenkins login.
+      - Optional. Default username for Jenkins login. Is not used in UI
```

Comment:
Line 136:

Update state of clusters on cluster list. This can be disabled for organizations with a large number of clusters in the deploy state.